packages/ChangesReloaded-Core.package/CRBrowser.class/instance/buildPanelSpecWith.andChildren..st

```diff
@@ -0,0 +1,7 @@
+toolbuilder
+buildPanelSpecWith: aBuilder andChildren: aCollectionOfSpecs
+
+	^ aBuilder pluggablePanelSpec new
+		 model: self;
+		 layout: #horizontal;
+		 children: aCollectionOfSpecs asOrderedCollection
```

Comment:
Isn't it an `OrderedCollection` already?

Reply:
Oh true, it was an `Array` before I refactored that.